ministryofjustice/analytics-platform-control-panel-frontend
# Remove login template and Auth0 Lock, switch to hosted login for SSO win

## What

That ^

## How to review

1. check out, do the docker down/pull/build/up dance probably
2. go to `/`
3. see that you are redirected to the "kubectl-oidc" login popup on Auth0 (titling and styling to come)
4. log in
5. see that you are returned to the control panel as usual
